app/assets/stylesheets/views.scss

```diff
@@ -0,0 +1,9 @@
+@import 'views/article';
```

Comment:
I'm wondering at some point in the future if we may want to rename this from "Article" to "Post". That seems like it would be more inline with how we think about this in our evolution. I know that the word "article" appears in a lot of places in our codebase, so we would want to do that carefully. Curious what @benhalpern @ludwiczakpawel @vaidehijoshi thinks about this random idea!